Resources/Prototypes/Nyanotrasen/Entities/Structures/Research/oracle.yml

```diff
@@ -16,7 +16,10 @@
   - type: Oracle
   - type: Speech
     speechSounds: Tenor
+  - type: PotentialPsionic
```

Comment:
Is this meant to be here?

Reply:
Yes, it's required for the Mantis to be able to use his scan power on Oracle, Metem Machine, and Sophie.